src/promptflow/promptflow/executor/flow_executor.py

```diff
@@ -793,15 +795,21 @@ def _stringify_generator_output(self, outputs: dict):
 
     def _submit_to_scheduler(self, context: FlowExecutionContext, inputs, nodes: List[Node]) -> Tuple[dict, dict]:
         if not isinstance(self._node_concurrency, int):
-            raise NodeConcurrencyNotFound("Need to set node concurrency as using flow executor.")
+            raise UnexpectedError(
+                message_format=(
+                    "Need to set valid node concurrency to run, current value is {current_value}. "
+                    "Please contact support for further assistance."
```

Comment:
Flow execution failed. To proceed, ensure that a valid node concurrency value is set; the current value is {current_value}. Please contact support for further assistance.